.github/matrix-full.yml

```diff
@@ -180,6 +180,12 @@ release-packages:
   artifact-version: 9
   artifact-type: rhel
   artifact: kong.el9.amd64.rpm
+- label: rhel-9-arm64
+  package: rpm
+  artifact-from: rhel-9-arm64
+  artifact-version: 9
+  artifact-type: rhel
+  artifact: kong.el9.arm64.rpm
 
 # Amazon Linux
 - label: amazonlinux-2
```

Comment:
Just for curious, I see amazon linux 2 supports arm64 as well (search key world "linux 2" on https://aws.amazon.com/ec2/graviton/). Will we add arm64 build for amazon linux 2?

Reply:
@zhongweiy yes we will do that eventually, the first step is to get arm64 support for latest version for each distro flavor, so we can give them a full test before rolling out more artifacts